lib/src/Timer/Timer.tsx

```diff
@@ -6,80 +6,67 @@ export type TimerProps = JSX.IntrinsicElements["div"] & {
     callback?: VoidFunction;
     /** Timer's duration in milliseconds */
     duration?: number;
+    /** Additional timer prefix */
+    timerPrefix?: React.ReactNode;
+    /** Additional timer suffix */
+    timerSuffix?: React.ReactNode;
 };
 
-interface TimerState {
-    timer: string;
-}
-/** A timer is a component for measuring time intervals */
-export class Timer extends React.Component<TimerProps, TimerState> {
-    private innerInterval: any;
-    constructor(props: TimerProps) {
-        super(props);
+export const Timer: React.FC<TimerProps> = ({ duration, callback, timerPrefix, timerSuffix, ...props }: TimerProps) => {
+    const [innerInterval, setInnerInterval] = React.useState<NodeJS.Timeout>();
+    const [timer, setTimer] = React.useState<string>("00:00");
 
-        this.state = {
-            timer: "00:00",
-        };
-    }
-
-    startInterval(timeout: number): void {
-        this.setState({ timer: this.convertMStoTime(timeout) }, () => {
-            this.clearInterval();
-            this.innerInterval = setInterval(() => {
+    const startInterval = (timeout: number): void => {
+        setTimer(convertMStoTime(timeout));
+        clearInterval();
```

Comment:
clear interval without interval id seem redundant, is this needed?

Reply:
should be clearInnerInterval